ibm_catalog.json

```diff
@@ -80,90 +81,133 @@
               ],
-              "notes": "Required to create and edit OpenShift cluster and the related resources."
+              "service_name": "iam-access-groups",
+              "notes": "[optional] Required for managing IAM access groups."
             },
             {
               "service_name": "iam-identity",
               "role_crns": [
-                "crn:v1:bluemix:public:iam::::role:Administrator",
+                "crn:v1:bluemix:public:iam::::role:Operator",
                 "crn:v1:bluemix:public:iam-identity::::serviceRole:UserApiKeyCreator"
               ],
-              "notes": "Required to create the containers-kubernetes-key needed by the OpenShift cluster on IBM Cloud."
+              "notes": "Required to create the containers-kubernetes-key for the OpenShift cluster and to manage resources within the IBM Cloud environment."
             },
             {
-              "service_name": "is.vpc",
               "role_crns": [
-                "crn:v1:bluemix:public:iam::::role:Administrator"
+                "crn:v1:bluemix:public:iam::::serviceRole:Manager",
+                "crn:v1:bluemix:public:iam::::role:Editor"
               ],
-              "notes": "Required for creating Virtual Private Cloud(VPC)."
+              "service_name": "hs-crypto",
+              "notes": "[Optional] Required if KMS encryption is enabled and Hyper Protect Crypto Service is used for encryption."
             },
             {
-              "service_name": "cloud-object-storage",
               "role_crns": [
                 "crn:v1:bluemix:public:iam::::serviceRole:Manager",
                 "crn:v1:bluemix:public:iam::::role:Editor"
               ],
-              "notes": "Required to create Cloud Object Storage (COS) Instance."
+              "service_name": "kms",
+              "notes": "[Optional] Required if KMS encryption is enabled and Key Protect Service is used for encryption."
             },
             {
-              "service_name": "hs-crypto",
               "role_crns": [
                 "crn:v1:bluemix:public:iam::::serviceRole:Manager",
                 "crn:v1:bluemix:public:iam::::role:Editor"
               ],
-              "notes": "[Optional] Required if KMS encryption is enabled and IBM Hyper Protect Crypto Services is used to encrypt the Kubernetes Secrets and Object Storage bucket."
+              "service_name": "cloud-object-storage",
+              "notes": "[Optional] Required to edit data and manage the OpenShift cluster's internal registry storage bucket."
             },
             {
-              "service_name": "kms",
+              "service_name": "containers-kubernetes",
               "role_crns": [
                 "crn:v1:bluemix:public:iam::::serviceRole:Manager",
+                "crn:v1:bluemix:public:iam::::role:Administrator"
+              ],
+              "notes": "Required to reset API keys, create and edit the OpenShift cluster, and manage all related resources."
```

Comment:
I think it's too detailed. 

eg: "Base permission requirement to create and manage the Openshift cluster"